src/common.rs

```diff
@@ -1,3 +1,45 @@
+/// A CPU architecture.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+#[allow(missing_docs)]
+pub enum Architecture {
+    Unknown,
+    Aarch64,
+    Arm,
+    I386,
+    Mips,
+    Wasm,
```

Comment:
```suggestion
    Wasm32,
```

There will eventually be wasm64 as well.

Reply:
Ok. I've changed this back to how target-lexicon has it (the address size is a property of the architecture). I think that works better for COFF too.